docs/reference/sql/limitations.asciidoc

```diff
@@ -161,3 +161,11 @@ By default,`geo_points` fields are indexed and have doc values. However only lat
 indexed with some loss of precision from the original values (4.190951585769653E-8 for the latitude and
 8.381903171539307E-8 for longitude). The altitude component is accepted but not stored in doc values nor indexed.
 Therefore calling `ST_Z` function in the filtering, grouping or sorting will return `null`.
+
+[float]
+[[fields-from-source]]
+=== Retrieving from _source
+
+Most of {es-sql}'s columns are retrieved from `_source`, even though the mapping may disable the <<mapping-source-field,`_source`>> field.
```

Comment:
This phrasing seems a bit confusing to me. I'd change this towards something like:
```
Most of {es-sql}'s columns are retrieved from `_source`, except from the fields of types: ..... 
``` (maybe list them with their SQL name instead of the ES name)
```If _source is disabled in the mapping and a column that must be retrieved from _source is requested in the SELECT .....```